src/react/boosts/BoostTextComponents.tsx

```diff
@@ -3,28 +3,45 @@ import { ModalHeading, ModalContent } from "@joincivil/components";
   </>
 );
 
 export const WhatIsEthModalText: React.FunctionComponent = props => (
   <>
-    <ModalHeading>
-      What is ETH?
-    </ModalHeading>
-    <ModalContent>
-      <p>Ether (ETH) is the cryptocurrency for the Ethereum blockchain. You’ll be paying in ETH to support and pay Boosts.</p>
-      <p>You can purchase or exchange for ETH using a cryptocurrency exchange such as Coinbase or Gemini and fund a digital wallet such as MetaMask. Don’t worry, both Coinbase and Gemini are regulated and in compliance with all applicable laws in each jurisdiction in which they operate. Buying ETH with a debit or certain credit cards is instant, once your account is verified.</p>
-      <p>Each transaction includes a small transaction cost, called gas, which is usually a few cents. These fees go to the Ethereum miners who perform the computational work for your content to be included on the Ethereum blockchain.</p>
-      <a href="">Learn more about how to get ETH and support Boosts on our FAQ.</a>
-    </ModalContent>
+    <ModalHeading>What is ETH?</ModalHeading>
+    <ModalContent>Ether (ETH) is the cryptocurrency for the Ethereum blockchain. You’ll be paying in ETH to support and pay Boosts.</ModalContent>
+    <ModalContent>You can purchase or exchange for ETH using a cryptocurrency exchange such as Coinbase or Gemini and fund a digital wallet such as MetaMask. Don’t worry, both Coinbase and Gemini are regulated and in compliance with all applicable laws in each jurisdiction in which they operate. Buying ETH with a debit or certain credit cards is instant, once your account is verified.</ModalContent>
+    <ModalContent>Each transaction includes a small transaction cost, called gas, which is usually a few cents. These fees go to the Ethereum miners who perform the computational work for your content to be included on the Ethereum blockchain.</ModalContent>
+    <ModalContent><a href="#TODO">Learn more about how to get ETH and support Boosts on our FAQ.</a></ModalContent>
+  </>
+);
+
+export const CanUseCVLText: React.FunctionComponent = props => (
+  <>
+    <ModalHeading>Can I use CVL to support a Boost?</ModalHeading>
+    <ModalContent>Civil tokens (CVL) are intended as a governance token to be used on the Civil Registry. You can use them to participate in and contribute to Civil’s community. Civil tokens unlock specific activities on the Civil platform, including launching a newsroom on the Registry, challenging and voting for/against Newsrooms for ethics violations or appealing the outcome of a community vote to the Civil Council.</ModalContent>
+    <ModalContent><a href="#TODO">Learn more about Civil tokens</a></ModalContent>
+  </>
+);
+
+export const PaymentInfoText: React.FunctionComponent = props => (
+  <>
+    <h3>Payment Information</h3>
+    <p>All Boosts procceded will be funded using ETH. Civil does not collect any fees on Boosts and your support goes directly to the Newsroom.</p>
```

Comment:
Ditto typo `procceded`, it's a pain but I'd check and copy paste all copy from invision.

Next time we should make sure to do a design+copy review before we start putting copy into code.